cli/CMakeLists.txt

```diff
@@ -10,10 +10,10 @@ list(REMOVE_ITEM srcs ${mainfile})
 add_library(cli_objs OBJECT ${hdrs} ${srcs}) 
 add_executable(cppcheck ${hdrs} ${mainfile} $<TARGET_OBJECTS:cli_objs> $<TARGET_OBJECTS:lib_objs> $<TARGET_OBJECTS:tinyxml_objs> $<TARGET_OBJECTS:simplecpp_objs>)
 if (HAVE_RULES)
-    find_library(PCRE_LIBRARY pcre)
     target_link_libraries(cppcheck ${PCRE_LIBRARY})
-    find_path(PCRE_INCLUDE pcre.h)
-    target_include_directories(cppcheck PUBLIC ${PCRE_INCLUDE})
+endif()
+if (USE_Z3)
+    target_link_libraries(cppcheck ${Z3_LIBRARY})
```

Comment:
This should link against the `z3::libz3` target provided by z3.

Reply:
See above.